source/agora/consensus/Fee.d

```diff
@@ -308,6 +308,9 @@ public class FeeManager
         tx_fees -= this.block_data_fees.get(height, 0.coins);
         tx_fees.percentage(this.params.ValidatorTXFeeCut);
 
+        // Add the block rewards to the Validator payouts
+        tx_fees.add(rewards.validator_rewards);
```

Comment:
**Always** check the return value of `add`. The other option is to use `a + b`, which throws an `Exception`. But **never** call `add`, `sub`, etc... without checking the return value.

Reply:
Updated and added another commit to fix a couple of existing cases.